lib/widgets/content.dart

```diff
@@ -816,17 +812,58 @@ class RealmContentNetworkImage extends StatelessWidget {
   }
 }
 
+/// A square showing a user's avatar.
+///
+/// To set the size and clip the corners to be round, pass [size].
+/// If [size] is not passed, the caller takes responsibility
+/// for doing that with its own square wrapper.
```

Comment:
It looks like the caller that's leaving out `size` is doing the same thing as this would, except with a different border radius. I think it's probably cleanest for this widget to keep responsibility for applying a size and shape, and just add a `double borderRadius` field that callers have to specify.

Reply:
Sure, I can do that.

On the other hand, that'll mean reciting the same size and border radius again for the group-DM icon/avatar, so there are two places to update if we want to change them. Or maybe I can put them in variables. 🤔